lisa/tools/kvm_unit_tests.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import string
+from pathlib import Path, PurePath
+from typing import List, Type, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+from .git import Git
+from .make import Make
+
+
+class KvmUnitTests(Tool):
```

Comment:
This tool is not used in other test suites ideally. So put it into the same folder as `microsoft/testsuites/kvm`.

Reply:
Shouldn't we have all tools in the "tools" directory?

Reply:
Done.